src/steps/pipeline/converter.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { CircleciPipeline, CircleciProject } from '../../types';
+
+export function getPipelineKey(id: string): string {
+  return `circleci_pipeline:${id.toString()}`;
+}
+
+export function getProjectKey(id: string): string {
+  return `circleci_project:${id.toString()}`;
+}
+
+export function createPipelineEntity(pipeline: CircleciPipeline): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: pipeline,
+      assign: {
+        _type: Entities.CONFIGURATION._type,
+        _class: Entities.CONFIGURATION._class,
+        _key: getPipelineKey(pipeline.id),
+        id: pipeline.id.toString(),
+        slug: pipeline.project_slug,
+        name: pipeline.id.toString(),
+      },
```

Comment:
Pipeline resources have `state` property that seems useful, I'd add it. Also it contains the created_at and updated_at timestamps, J1 SDK has function that parses that and return unix in milliseconds.

```suggestion
        id: pipeline.id.toString(),
        slug: pipeline.project_slug,
        name: pipeline.id.toString(),
        state: pipeline.state,
        createdOn: parseTimePropertyValue(pipeline.created_at),
        updatedOn: parseTimePropertyValue(pipeline.updated_at),
      },
```